bootstrap/aws-public/config-default.sh

```diff
@@ -19,6 +19,8 @@ export TF_VAR_master_size=${TF_VAR_master_size:-m1.medium}
 export TF_VAR_slave_size=${TF_VAR_slave_size:-m1.medium}
 export TF_VAR_slaves=${TF_VAR_slaves:-1}
 export TF_VAR_subnet_availability_zone=${TF_VAR_subnet_availability_zone:-eu-west-1a}
+# Overrides default folder in Terraform.py inventory.
+export TFSTATE_ROOT=`pwd`/terraform/aws-public
```

Comment:
this cant be the pwd of this script? is it better to make it relative to $APOLLO_ROOT?